### ISSUE TITLE
jinput error

### ISSUE THREAD

**Issue by kephale** [scenerygraphics/sciview#271]
When testing `VolumeRenderDemo` on commit cf953368560b836e076e273de0c4328277958abd with mac os:

```
java.lang.UnsatisfiedLinkError: Can't load library: /var/folders/0k/x8yqjgps1ms9xr7ndbswclgm0000gn/T/scenery-natives-tmp2717343882908347840/libjinput-osx.dylib
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1827)
	at java.lang.Runtime.load0(Runtime.java:809)
	at java.lang.System.load(System.java:1086)
	at net.java.games.input.OSXEnvironmentPlugin.lambda$loadLibrary$0(OSXEnvironmentPlugin.java:68)
	at java.security.AccessController.doPrivileged(Native Method)
	at net.java.games.input.OSXEnvironmentPlugin.loadLibrary(OSXEnvironmentPlugin.java:64)
	at net.java.games.input.OSXEnvironmentPlugin.<clinit>(OSXEnvironmentPlugin.java:93)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at net.java.games.input.DefaultControllerEnvironment.getControllers(DefaultControllerEnvironment.java:133)
	at graphics.scenery.controls.MouseAndKeyHandlerBase.<init>(MouseAndKeyHandlerBase.kt:147)
	at graphics.scenery.controls.SwingMouseAndKeyHandler.<init>(SwingMouseAndKeyHandler.kt:52)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at graphics.scenery.controls.InputHandler.<init>(InputHandler.kt:71)
	at graphics.scenery.controls.InputHandler.<init>(InputHandler.kt:32)
	at graphics.scenery.SceneryBase.loadInputHandler(SceneryBase.kt:441)
	at graphics.scenery.SceneryBase.sceneryMain$suspendImpl(SceneryBase.kt:217)
	at graphics.scenery.SceneryBase.sceneryMain(SceneryBase.kt)
	at graphics.scenery.SceneryBase$main$1.invokeSuspend(SceneryBase.kt:483)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:272)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:79)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:54)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:36)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
	at graphics.scenery.SceneryBase.main(SceneryBase.kt:483)
	at sc.iview.DefaultSciViewService.lambda$makeSciView$0(DefaultSciViewService.java:114)
	at org.scijava.thread.DefaultThreadService.lambda$wrap$1(DefaultThreadService.java:214)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

Current related issue: https://forum.image.sc/t/bug-in-3d-viewer-record-and-background-change-how-to-use-sciview/37923/9

**Comment by skalarproduktraum** [comment]
@imagesc-bot, you are very useful, thank you!

Also, the issue in the forum is a false positive, with the error just popping out because of a debug message. The library is actually found and loaded.

@kephale's problem however is different and pertains to the change of library extensions from jnilib to dylib. This is addressed in https://github.com/scenerygraphics/scenery/pull/317

**Comment by skalarproduktraum** [comment]
Fixed as of scenerygraphics/scenery#317. Please open a new issue should this resurface.
